Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "macosx-v3.3.tar.gz" "fakeroot" "host-fakeroot" "include-guid-type" "\
        patchattr.h \
    ";


===============================================================================
--- a/patchattr.h	2012-05-24 11:23:04.000000000 +0200
+++ b/patchattr.h	2018-01-03 19:44:09.000000000 +0100
@@ -14,6 +14,7 @@
 
 #include <sys/attr.h>
 #include <sys/mount.h>
+#include <sys/_types/_guid_t.h>
 
 #ifndef ATTR_CMN_FILEID
 #define ATTR_CMN_FILEID                         0x02000000
